corehq/motech/repeaters/management/commands/list_repeater_domain_admins.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import logging
+from argparse import FileType
+
+from django.core.management.base import BaseCommand
+from corehq.apps.users.models import WebUser
+from corehq.motech.repeaters.models import Repeater
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """Shows names and admin emails of Domains with Repeaters."""
+
+    def add_arguments(self, parser):
+        parser.add_argument("--csv", action="store_true", default=False,
+            help="Write output as CSV data instead of padded table.")
+        parser.add_argument("-o", "--output", metavar="FILE", type=FileType("w"),
+            default=self.stdout, help="Write output to %(metavar)s (default=STDOUT).")
+
+    def handle(self, *args, **options):
+        log.debug("fetching repeaters")
+        table = Table(["Domain", "Admins"])
+        for summary in Repeater.get_db().view('repeaters/repeaters',
+                                              group_level=1, reduce=True).all():
+            domain = summary["key"][0]
+            log.debug("domain %s has %s repeater(s)", domain, summary['value'])
+            emails = set()
+            for admin in WebUser.get_admins_by_domain(domain):
+                emails.add(admin.email if admin.email else admin.username)
+            table.add_row([domain, ", ".join(sorted(emails))])
+        table.sort()
+        if options["csv"]:
+            table.write_csv(options["output"])
+        else:
+            options["output"].write(table.render())
+
+
+class Table:
```

Comment:
Probably unreasonable to add a new requirement just for one management command, but I love [tablib](https://pypi.org/project/tablib/).

```python
>>> data = tablib.Dataset(headers=['First Name', 'Last Name', 'Age'])
>>> for i in [('Kenneth', 'Reitz', 22), ('Bessie', 'Monke', 21)]:
...     data.append(i)

>>> data.export('df')
  First Name Last Name  Age
0    Kenneth     Reitz   22
1     Bessie     Monke   21

>>> print(data.export('rst'))                                                            
+---------+-------+-----+
|  First  |  Last | Age |
|   Name  |  Name |     |
+=========+=======+=====+
| Kenneth | Reitz | 22  |
+---------+-------+-----+
| Bessie  | Monke | 21  |
+---------+-------+-----+

>>> print(data.export('csv'))                                                                                                   
First Name,Last Name,Age
Kenneth,Reitz,22
Bessie,Monke,21

```
(I think I forgot to click "Start a review" and I lost this comment, so trying again. Sorry if you got notified of the same comment twice.)